modules/qualitycontrol/ReceiveVideoController.js

```diff
@@ -0,0 +1,86 @@
+
+import * as JitsiConferenceEvents from '../../JitsiConferenceEvents';
+
+/**
+ * This class manages the receive video contraints for a given {@link JitsiConference}. These constraints are
+ * determined by the application based on how the remote video streams need to be displayed. This class is responsible
+ * for communicating these constraints to the bridge over the bridge channel.
+ */
+export class ReceiveVideoController {
+    /**
+     * Creates a new instance for a given conference.
+     *
+     * @param {JitsiConference} conference the conference instance for which the new instance will be managing
+     * the receive video quality constraints.
+     * @param {RTC} rtc the rtc instance which is responsible for initializing the bridge channel.
+     */
+    constructor(conference, rtc) {
+        this.conference = conference;
```

Comment:
It's better to store both `conference` and `rtc` references with the underscore notations, so the are kind of private. I know this is not consistent, but lets keep at least new stuff the right way.